webview/src/setup/components/App.tsx

```diff
@@ -58,76 +56,28 @@ export const App: React.FC = () => {
     )
   )
 
-  const checkCompatibility = () => {
-    sendMessage({ type: MessageFromWebviewType.CHECK_CLI_COMPATIBLE })
-  }
-
-  const initializeGit = () => {
-    sendMessage({
-      type: MessageFromWebviewType.INITIALIZE_GIT
-    })
-  }
-
-  const initializeDvc = () => {
-    sendMessage({
-      type: MessageFromWebviewType.INITIALIZE_DVC
-    })
-  }
-
-  const showScmPanel = () => {
-    sendMessage({ type: MessageFromWebviewType.SHOW_SCM_PANEL })
-  }
-
-  const installDvc = () => {
-    sendMessage({ type: MessageFromWebviewType.INSTALL_DVC })
-  }
-
-  const selectPythonInterpreter = () => {
-    sendMessage({ type: MessageFromWebviewType.SELECT_PYTHON_INTERPRETER })
-  }
-
-  const setupWorkspace = () => {
-    sendMessage({ type: MessageFromWebviewType.SETUP_WORKSPACE })
-  }
-
-  if (cliCompatible === false) {
-    return <CliIncompatible checkCompatibility={checkCompatibility} />
-  }
-
-  if (cliCompatible === undefined) {
-    return (
-      <CliUnavailable
-        installDvc={installDvc}
-        isPythonExtensionInstalled={isPythonExtensionInstalled}
-        pythonBinPath={pythonBinPath}
-        selectPythonInterpreter={selectPythonInterpreter}
-        setupWorkspace={setupWorkspace}
-      />
-    )
-  }
-
-  if (!projectInitialized) {
-    return (
-      <ProjectUninitialized
+  return (
+    <SectionContainer
+      sectionCollapsed={sectionCollapsed[Section.EXPERIMENTS]}
+      sectionKey={Section.EXPERIMENTS}
+      title={'Experiments'}
+      onToggleSection={() =>
+        setSectionCollapsed({
+          ...sectionCollapsed,
+          [Section.EXPERIMENTS]: !sectionCollapsed[Section.EXPERIMENTS]
+        })
+      }
+    >
+      <SetupExperiments
         canGitInitialize={canGitInitialize}
-        initializeDvc={initializeDvc}
-        initializeGit={initializeGit}
+        cliCompatible={cliCompatible}
+        hasData={hasData}
+        isPythonExtensionInstalled={isPythonExtensionInstalled}
         needsGitInitialized={needsGitInitialized}
+        needsGitCommit={needsGitCommit}
+        projectInitialized={projectInitialized}
+        pythonBinPath={pythonBinPath}
```

Comment:
[Q] Should I just match the plots and experiments `<App/>`s and bring redux in here now?

Reply:
I don't think it's worth it yet for the few levels that we have and how simple the components are.